src/plugins/Webcam/index.js

```diff
@@ -330,7 +330,9 @@ module.exports = class Webcam extends Plugin {
       recording: webcamState.isRecording,
       mirror: this.opts.mirror,
       src: this.stream
-    }))
+    })
+
+    return <CameraScreen {...CameraScreenProps} />
```

Comment:
We could also only use JSX spread here instead of object.assign:

```js
<CameraScreen
  {...webcamState}
  onSnapshot={this.takeSnapshot}
  /* ... etc ... */
  src={this.stream}
/>
```